pkg/networkutils/network_test.go

```diff
@@ -838,6 +868,82 @@ func TestUpdateHostIptablesRules(t *testing.T) {
 			},
 		}, mockIptables.(*mock_iptables.MockIptables).DataplaneState)
 }
+
+func TestCleanUpStaleAWSChains(t *testing.T) {
+	ctrl, mockNetLink, _, mockNS, mockIptables := setup(t)
+	defer ctrl.Finish()
+
+	ln := &linuxNetwork{
+		useExternalSNAT:        false,
+		ipv6EgressEnabled:      true,
+		nodePortSupportEnabled: true,
+		mainENIMark:            defaultConnmark,
+		mtu:                    testMTU,
+		vethPrefix:             eniPrefix,
+
+		netLink: mockNetLink,
+		ns:      mockNS,
+		newIptables: func(iptables.Protocol) (iptableswrapper.IPTablesIface, error) {
+			return mockIptables, nil
+		},
+	}
+	setupNetLinkMocks(ctrl, mockNetLink)
+
+	_ = mockIptables.NewChain("nat", "AWS-SNAT-CHAIN-1")
```

Comment:
nit: no need to even try to capture the return value with `_ =` if it won't be used. You can just call the function with `mockIptables.NewChain("nat", "AWS-SNAT-CHAIN-1")`